pkg/core/mpt/compat_test.go

```diff
@@ -72,10 +72,11 @@ func TestCompatibility(t *testing.T) {
 
 		require.Equal(t, mainTrie.root.Hash(), tr.root.Hash())
 		require.Error(t, tr.Put(nil, []byte{0x01}))
-		require.NoError(t, tr.Put([]byte{0x01}, nil))
```

Comment:
These tests are also altered in C# PR, we need to follow. For example this line should be changed to
`require.NoError(t, tr.Put([]byte{0x01}, []byte{}))`
https://github.com/neo-project/neo-modules/pull/634/files#diff-a7cab0a54353f7147388e535326be4adc8bd3ba2c26e0b48d54ab049566359cfR197

Reply:
It is already replaced, but a few lines below (after the check that root hash of the etalon tree is equal to the actual one). In C# they also do not change the etalon tree and perform addition of the empty value after this check.

Reply:
As a result, this line checks that empty value can be safely added to the tree, but do not perform root hashes comparison. C# code does the same.